crowdin_api/api_resources/security_logs/resource.py

```diff
@@ -0,0 +1,114 @@
+from typing import Optional
+
+from crowdin_api.api_resources.abstract.resources import BaseResource
+from crowdin_api.api_resources.security_logs.enums import SecurityLogEvent
+
+
+class BaseSecurityLogsResource(BaseResource):
+    def get_user_security_logs_path(
+        self, userId: int, securityLogId: Optional[int] = None
+    ):
+        if securityLogId is not None:
+            return f"/users/{userId}/security-logs/{securityLogId}"
+        return f"/users/{userId}/security-logs"
+
+    def list_user_security_logs(
+        self,
+        userId: int,
+        limit: Optional[int] = None,
+        offset: Optional[int] = None,
+        page: Optional[int] = None,
+        event: Optional[SecurityLogEvent] = None,
+        ipAddress: Optional[str] = None,
+    ):
+        """
+        List User Security Logs
+
+        Link to documentaion:
+        https://developer.crowdin.com/api/v2/#operation/api.users.security-logs.getMany
+        """
+
+        params = {"event": event, "ipAddress": ipAddress}
+        params.update(self.get_page_params(page=page, offset=offset, limit=limit))
+
+        return self._get_entire_data(
+            method="get",
+            path=self.get_user_security_logs_path(userId=userId),
+            params=params,
+        )
+
+    def get_user_security_log(self, userId: int, securityLogId: int):
+        """
+        Get User Security Log
+
+        Link to documentation:
+        https://developer.crowdin.com/api/v2/#operation/api.users.security-logs.get
+        """
+
+        return self.requester.request(
+            method="get",
+            path=self.get_user_security_logs_path(
+                userId=userId, securityLogId=securityLogId
+            ),
+        )
+
+
+class SecurityLogsResource(BaseSecurityLogsResource):
+    """
+    Resource for Security Logs
+
+    Link to documentaion:
+    https://developer.crowdin.com/api/v2/#tag/Security-Logs
+    """
+
+
+class EnterpriseSecurityLogsResource(BaseSecurityLogsResource):
```

Comment:
I think this is overhead, we don't really need a separate resource. Just keep all the methods in a single resource